### PR TITLE
Mar 908

### PR DESCRIPTION
**Pull request by jamespacileo**

### CODE REVIEW

File: core/frontend/src/css/pages/barrier/_detail.scss [hunk 1]
```diff
@@ -40,6 +40,62 @@
...
+			align-items: center;
+			position: relative;
+			// height: 60px;
+
+			.checkbox-wrapper{
+				position: relative;
+				margin: 4px;
+			}
+
+			// .govuk-checkboxes__input{
```
Comment (cad106uk) [inline core/frontend/src/css/pages/barrier/_detail.scss:82]: Delete the commented out code.

**Review by cad106uk — CHANGES_REQUESTED** [review]
Looks good, needs a couple of changes.

File: core/frontend/src/css/pages/barrier/_detail.scss (hunk 1 above)
Comment (jamespacileo) [reply core/frontend/src/css/pages/barrier/_detail.scss:82]: removed commented out code

File: barriers/models/barriers.py [hunk 2]
```diff
@@ -461,3 +467,25 @@ def tab_badge(self):
...
+    def awaiting_reviews_from(self):
+        from_who = []
+        if not self.light_touch_reviews["content_team_approval"]:
+            from_who.append("Content")
+        if (
+            self.light_touch_reviews["hm_trade_commissioner_approval_enabled"]
+            and not self.light_touch_reviews["hm_trade_commissioner_approval"]
+        ):
+            from_who.append("HM Trade Commissioner")
+        for org in self.internal_government_organisations:
```
[the remark this answers is not in the document]
Comment (jamespacileo) [reply barriers/models/barriers.py:485]: didn't realise you could do it that way, thanks!

File: barriers/models/barriers.py [hunk 3]
```diff
@@ -328,6 +328,12 @@ def internal_government_organisations(self):
     def internal_government_organisations_names(self):
         return [org["name"] for org in self.internal_government_organisations]
 
+    @property
+    def internal_government_organisations_lookup(self):
+        return dict(
```
[the remark this answers is not in the document]
Comment (jamespacileo) [reply barriers/models/barriers.py]: removed the method as it's not actually being used eeek

but fair enough, guess the assumption is that the code is unique

**Review by cad106uk — APPROVED** [review]

**Pull request merged**